programmers/다트-게임.js

```diff
@@ -0,0 +1,57 @@
+function solution(dartResult) {
+    let num = 0;
+    const answer = [];
+    let temp = 0;
```

Comment:
```suggestion
    const answer = [];
    let num = 0;
    let temp = 0;
```
가독성을 위해 const를 위에 써주고 그다음 let 써주는게 좋을 것 같아요ㅎㅎ